src/lib.rs

```diff
@@ -115,3 +121,13 @@ pub fn set_times(config: Config) {
         thread::sleep(Duration::from_secs(60));
     }
 }
+
+fn get_current_wallpaper(wall_len: usize) -> usize {
```

Comment:
Just commend on what this function does.

Reply:
This is resolved in the latest commit.